src/Core.UnitTests/Notifications/DisabledNotificationsStorageTests.cs

```diff
@@ -36,7 +36,10 @@ namespace SonarLint.VisualStudio.Core.UnitTests.Notifications
     [TestClass]
     public class DisabledNotificationsStorageTests
     {
-        private string ExpectedDisabledNotificationsFilePath => GetExpectedDisabledNotificationsFilePath();
+        private const string DefaultRootPath = "c:\\users\\someuser";
+        private const string HardCodedVSVersion = "17";
```

Comment:
Miscellaneous: `17` was a magic number that was used in a couple of separate-but-related methods. I've made it a constant to make the connection between the locations in code a little bit more obvious.